packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -423,12 +400,19 @@ class WebSettings {
 /// See [WebView.addJavaScriptChannel].
 abstract class JavaScriptChannel {
   /// Constructs a [JavaScriptChannel].
-  JavaScriptChannel(this.channelName);
+  JavaScriptChannel(this.channelName) {
+    if (!_flutterApisHaveBeenSetup) {
+      JavaScriptChannelFlutterApi.setup(JavaScriptChannelFlutterApiImpl());
+      _flutterApisHaveBeenSetup = true;
+    }
+  }
 
   /// Pigeon Host Api implementation for [JavaScriptChannel].
   @visibleForTesting
   static JavaScriptChannelHostApiImpl api = JavaScriptChannelHostApiImpl();
 
+  static bool _flutterApisHaveBeenSetup = false;
```

Comment:
Nit: `_flutterApisHaveBeenSetUp`. (Our code shouldn't use the wrong name even though Pigeon does.)

Reply:
Done